Formatter/Pool.php

```diff
@@ -34,15 +34,13 @@ class Pool
     protected $logger;
 
     /**
-     * @param LoggerInterface|null $logger
      * @param string|null          $defaultFormatter
+     * @param LoggerInterface|null $logger
      */
-    public function __construct(LoggerInterface $logger = null, $defaultFormatter = null)
+    public function __construct($defaultFormatter, LoggerInterface $logger = null)
```

Comment:
Maybe also mention that passing null will no longer automatically make the first formatter the default one?